src/Uno/Prism.Uno/Prism.Uno.csproj

```diff
@@ -0,0 +1,85 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;xamarinios10;monoandroid90;xamarinmac20;uap10.0.16299</TargetFrameworks>
+    <GenerateLibraryLayout>true</GenerateLibraryLayout>
+    <RootNamespace>Prism</RootNamespace>
+    <DefineConstants>$(DefineConstants);HAS_WINUI</DefineConstants>
+    <NoWarn>$(NoWarn);1591</NoWarn>
+    <PackageId>Uno.Prism.Uno</PackageId>
```

Comment:
This package id should be Prism.Uno

Reply:
> In general looks really good... needs a couple of minor changes... as discussed on stream.
> 
> If we can support XmlnsDefinitions to do `xmlns:prism="http://prismlibrary.com"` that would be great

Xmlns definitions is a missing feature from WinUI, and Uno sticks to that feature set. Maybe WinUI 3.1 will enable that :)

> Also noticed we have outputs in the artifacts for the WASM UI Tests but not iOS/Android

I'll normalize the outputs if needed, but there's an azure devops issue from the past few days that prevents attachments to be be uploaded properly when running on macOS. I'll reference it here when created.

> If you can normalize spaces to 4 spaces instead of mixing tabs/spaces
Definitely will. Mixup comes from the Uno templates that are tab based.